CHANGELOG.md

```diff
@@ -6,26 +6,34 @@ accidentally triggering the load of a previous DB version.**
 
 ## Unreleased
 
+## 2.10.0 (2023-02-16)
+
 **Features**
+* #4874 Allow joins in continuous aggregates
+* #4926 Refactor INSERT into compressed chunks
 * #5241 Allow RETURNING clause when inserting into compressed chunks
-* #5245 Mange life-cycle of connections via memory contexts
+* #5243 Enable real-time aggregation for continuous aggregates with joins
+* #5245 Manage life-cycle of connections via memory contexts
 * #5246 Make connection establishment interruptible
 * #5253 Make data node command execution interruptible
-* #5243 Enable real-time aggregation for continuous aggregates with joins
 * #5262 Extend enabling compression on a continuous aggregrate with 'compress_segmentby' and 'compress_orderby' parameters
 
 **Bugfixes**
-* #4926 Fix corruption when inserting into compressed chunks
-* #5218 Add role-level security to job error log
 * #5214 Fix use of prepared statement in async module
-* #5290 Compression can't be enabled on continuous aggregates when segmentby/orderby columns need quotation
+* #5218 Add role-level security to job error log
 * #5239 Fix next_start calculation for fixed schedules
+* #5290 Fix enabling compression on continuous aggregates with columns requiring quotation
+
++**Thanks**
```

Comment:
```suggestion
**Thanks**
```

There is a `+` at the beginning of the line (hard to see in the diff view on GitHub).